frameworks/cassandra/tests/config.py

```diff
@@ -84,7 +80,7 @@ def _get_test_job(
         "id": "test.cassandra." + name,
         "run": {
             "cmd": " && ".join(commands),
-            "docker": {"image": "cassandra:3.0.13"},
+            "docker": {"image": "cassandra:3.11.3"},
```

Comment:
let's introduce a variable for Cassandra version

Reply:
Ok will update that.